src/services/chris_integration.tsx

```diff
@@ -239,6 +251,16 @@ class ChrisIntegration {
     return status === PluginPollStatus.SUCCESS || status === PluginPollStatus.ERROR || status === PluginPollStatus.CANCELLED;
   }
 
+  static async getPluginData(id: number): Promise<pluginData> {
+    const client: any = ChrisAPIClient.getClient();
```

Comment:
Please, avoid using `any` since defeats the purpose of TypeScript. Use `any` only when the value can actually be any type.

```suggestion
    const client: Client = ChrisAPIClient.getClient();
```